cypress/support/accessibility/a11y-utils.js

```diff
@@ -114,6 +114,7 @@ export default (from, end) => {
       !prepareUrl[0].startsWith("popover-container") &&
       !prepareUrl[0].startsWith("menu") &&
       !prepareUrl[0].startsWith("inline-inputs") &&
+      !prepareUrl[0].startsWith("tile") &&
```

Comment:
should this also include:
      !prepareUrl[0].startsWith("tile-select") &&

Reply:
no, becase `tile-select` as well starts with `tile` :)